CMakeLists.txt

```diff
@@ -2,13 +2,20 @@ cmake_minimum_required(VERSION 3.12)
 project(tmp LANGUAGES CXX)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/")
+
+include(GNUInstallDirs)
```

Comment:
Shouldn't this be added under the `CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR` condition?